docs/getting-started/getting-started-with-galaxy.rst

```diff
@@ -53,6 +55,13 @@ outputs, and Galaxy Tool Shed dependencies for your workflow.
 
    Download
 
+If you've installed `Planemo <https://planemo.readthedocs.io/en/latest/index.html>`__, generate a ``.dockstore.yml`` file according to the `IWC instructions <https://github.com/galaxyproject/iwc/blob/main/workflows/README.md>`__, by navigating to the directory containing your workflow and run:
+
+::
+
+$ planemo dockstore_init .
+
+The ``.dockstore.yml`` is used in conjunction with the :doc:`Dockstore GitHub App <github-apps/github-apps-landing-page>`. For other workflow languages, the ``.dockstore.yml`` is created in a text editor; for Galaxy you have the option of generating with a command-line tool, and it's recommended you do so.
```

Comment:
This sentence feels like it should come before `planemo dockstore_init` since it explains why you want to generate one using planemo.